css/styles.css

```diff
@@ -1032,6 +1025,39 @@ a {
     height: 24px;
   }
 
+  .primary-buttons .button1 {
+    background-color: #0b37a8;
+    color: #fff;
+  }
+

```

Comment:
- [ ] Like the previous reviewer mentioned please fix the spacing around your design 🙏 . This might be as a result of the `width` and `margin` give to the `elements`

Desktop Design | Mobile Design
------------- | -------------
![20220719-222409_capture](https://user-images.githubusercontent.com/74814780/179851883-cfad4cf8-6b18-4f90-8b82-68f7469ecc0a.gif)  |  ![20220719-222615_capture](https://user-images.githubusercontent.com/74814780/179851977-af46efe1-37f6-4c2f-bcad-1f40eef22e66.gif)